docs/adding games.md

```diff
@@ -1,269 +1,76 @@
-
-### Requirements
-
-A world implementation requires a few key things from its implementation
-
-- A folder within `worlds` that contains an `__init__.py`
-    - This is what defines it as a Python package and how it's able to be imported
-      into Archipelago's generation system. During generation time only code that is
-      defined within this file will be run. It's suggested to split up your information
-      into more files to improve readability, but all of that information can be
-      imported at its base level within your world.
-- A `World` subclass where you create your world and define all of its rules
-  and the following requirements:
-    - Your items and locations need a `item_name_to_id` and `location_name_to_id`,
-      respectively, mapping.
-    - An `option_definitions` mapping of your game options with the format
-      `{name: Class}`, where `name` uses Python snake_case.
-    - You must define your world's `create_item` method, because this may be called
-      by the generator in certain circumstances
-    - When creating your world you submit items and regions to the Multiworld.
-        - These are lists of said objects which you can access at
-          `self.multiworld.itempool` and `self.multiworld.regions`. Best practice for
-          adding to these lists is with either `append` or `extend`, where `append` is a
-          single object and `extend` is a list.
-        - Do not use `=` as this will delete other worlds' items and regions.
-        - Regions are containers for holding your world's Locations.
-        - Locations are where players will "check" for items and must exist within
-          a region. It's also important for your world's submitted items to be the same as
-          its submitted locations count.
-        - You must always have a "Menu" Region from which the generation algorithm
-          uses to enter the game and access locations.
-- Make sure to check out [world maintainer.md](/docs/world%20maintainer.md) before publishing.
\ No newline at end of file
+# Adding Games
+
+Adding a new game to Archipelago has two major parts:
+
+* Game Modification to communicate with Archipelago server (hereafter referred to as "client")
+* Archipelago Generation and Server integration plugin (hereafter referred to as "world")
+
+This document will attempt to illustrate the bare minimum requirements and expectations of both parts of a new world
+integration. As game modification wildly varies by the system and engine and has no bearing on the Archipelago protocol
+it will not be detailed here.
+
+## Client
+
+The client is an intermediary program between the game and the Archipelago server. This can either be a direct
+modification to the game, an external program, or both. This can be implemented in nearly any modern language, but it
+must fulfill a few requirements in order to function as expected. The specific requirements for the game client in
+order to behave as expected are as follows:
```

Comment:
"in order" should be removed here to change the sentence to "The specific requirements for the game client to behave as expected are as follows:"